public/components/page-dashboard/client-projects/client-projects.stache

```diff
@@ -14,88 +14,88 @@
-			{{#if monthlyClientProject.clientProjectRef.value}}
+		{{# each(contributionMonth.sortedMonthlyClientProjects, monthlyClientProject=value) }}
+			{{# if(monthlyClientProject.clientProjectRef.value) }}
 				<tr>
-					<td>{{monthlyClientProject.clientProjectRef.value.name}}</td>
+					<td>{{ monthlyClientProject.clientProjectRef.value.name }}</td>
 					<td>
 						<input type="number" class="form-control hours-input"
 							value:bind="monthlyClientProject.hours"
-							on:change="contributionMonth.save()"
-							disabled:from="or(contributionMonth.isSaving(), notAdmin())"
+							on:change="../contributionMonth.save()"
+							disabled:from="or(../contributionMonth.isSaving(), notAdmin())"
 						/>
 					</td>
 					<td class="os-projects-cell">
-						<button type="button" class="btn btn-link btn-sm" data-toggle="modal" data-target="#os-project-modal" on:click="setSelectedClientProject(monthlyClientProject)" >
+						<button type="button" class="btn btn-link btn-sm" data-toggle="modal" data-target="#os-project-modal" on:click="../setSelectedClientProject(monthlyClientProject)" >
 							{{! Count of commissioned projects }}
-							{{contributionMonth.commissionedMonthlyOSProjectsCountFor(monthlyClientProject)}} / {{contributionMonth.monthlyOSProjects.commissioned.length}}
+							{{ scope.vm.contributionMonth.commissionedMonthlyOSProjectsCountFor(monthlyClientProject) }} / {{ scope.vm.contributionMonth.monthlyOSProjects.commissioned.length }}
 							+
 							{{! Count of uncommisioned projects }}
-							{{contributionMonth.uncommissionedMonthlyOSProjectsCountFor(monthlyClientProject)}}
+							{{ scope.vm.contributionMonth.uncommissionedMonthlyOSProjectsCountFor(monthlyClientProject) }}
 						</button>
 					</td>
 					<td>
-						 <span class="rate">${{contributionMonth.getRate(monthlyClientProject)}}</span>
+						 <span class="rate">${{ scope.vm.contributionMonth.getRate(monthlyClientProject) }}</span>
 					</td>
 					<td class="text-right">
-						<span class="total">${{contributionMonth.getTotal(monthlyClientProject).toFixed(2)}}</span>
+						<span class="total">${{ scope.vm.contributionMonth.getTotal(monthlyClientProject).toFixed(2) }}</span>
 					</td>
 					<td>
-						{{#if isAdmin()}}
-							<button type="button" class="btn btn-danger btn-xs" on:click="deleteClientProject(monthlyClientProject)">
+						{{# if(isAdmin()) }}
+							<button type="button" class="btn btn-danger btn-xs" on:click="../deleteClientProject(monthlyClientProject)">
 								<span class="glyphicon glyphicon-remove"/>
 							</button>
-						{{/if}}
+						{{/ if }}
 					</td>
 				</tr>
-			{{/if}}
-		{{/each}}
+			{{/ if }}
+		{{/ each }}
 		</tbody>
 	</table>
 
-	<can-import from="./client-projects-os-projects-modal.stache" @value:to="@*osProjectsModal" />
-	{{>*osProjectsModal}}
-{{/if}}
+	<can-import from="./client-projects-os-projects-modal.stache" value:to="scope.vars.osProjectsModal" />
+	{{ >scope.vars.osProjectsModal }}
```

Comment:
Instead of template variables using `*` as in `*osProjectsModal`, template variables are accessible in `scope.vars` like `scope.vars.osProjectsModal`.  Also, there's no longer a need to use `@` to prevent reading functions.